test/scripts/util/useragent.js

```diff
@@ -0,0 +1,13 @@
+var assert = require('assert'),
+  pkg = require('../../../package.json'),
+  ua = require('../../../scripts/util/useragent');
+
+describe('useragent', function() {
```

Comment:
I'd probably keep this file at the root of `test` and just use other levels of `describe` if you're trying to show depth (for now)